fairmq/shmem/Monitor.h

```diff
@@ -37,7 +47,19 @@ class Monitor
     void CatchSignals();
     void Run();
 
-    static void Cleanup(const std::string& shmId);
+    /// @brief Cleanup all shared memory artifacts created by devices
```

Comment:
@ktf Is this in line with what you have requested? `Cleanup()` should suffice if you never launch the shmmonitor, otherwise `CleanupFull()` will remove any artifacts created by device/monitor.

Reply:
Yes, I suspect this is what I want. I never create the shmonitor right now.